includes/wc-stock-functions.php

```diff
@@ -52,6 +52,11 @@ function wc_update_product_stock( $product, $stock_quantity = null, $operation =
 			do_action( 'woocommerce_product_set_stock', $product_with_stock );
 		}
 
+		// Send low stock notification for variable products
```

Comment:
We don't need this section of code IMO - it's handled elsewhere. `wc_trigger_stock_change_notifications`